ingest/workflow/snakemake_rules/fetch_sequences.smk

```diff
@@ -0,0 +1,135 @@
+"""
+This part of the workflow handles fetching sequences from various sources.
+Uses `config.sources` to determine which sequences to include in final output.
+
+Currently only fetches sequences from GenBank, but other sources can be
+defined in the config. If adding other sources, add a new rule upstream
+of rule `fetch_all_sequences` to create the file `data/{source}.ndjson` or the
+file must exist as a static file in the repo.
+
+Produces final output as
+
+    sequences_ndjson = "data/sequences.ndjson"
+
+"""
+
+
+rule fetch_ncbi_dataset_package:
+    output:
+        dataset_package=temp("data/ncbi_dataset.zip"),
+    retries: 5  # Requires snakemake 7.7.0 or later
+    benchmark:
+        "benchmarks/fetch_ncbi_dataset_package.txt"
+    shell:
+        """
+        datasets download virus genome taxon 12637 \
```

Comment:
The taxon id could be factored out into config

Reply:
Agree, submitted a PR to monkeypox https://github.com/nextstrain/monkeypox/pull/217
Will rebase those changes here after it's approved and merged.